bin/brew

```diff
@@ -84,7 +84,18 @@ HOMEBREW_LIBRARY="${HOMEBREW_REPOSITORY}/Library"
 
 # Copy and export all HOMEBREW_* variables previously mentioned in
 # manpage or used elsewhere by Homebrew.
-for VAR in BAT_THEME BROWSER DISPLAY EDITOR NO_COLOR TMUX DBUS_SESSION_BUS_ADDRESS XDG_RUNTIME_DIR
+PASSTHROUGH_VARS=(
+  BAT_THEME
+  BROWSER
+  DBUS_SESSION_BUS_ADDRESS
+  DISPLAY
+  EDITOR
+  LD_LIBRARY_PATH
```

Comment:
We should backup and then unset `LD_LIBRARY_PATH` at the top of `bin/brew`. May be insert at line 29:

https://github.com/Homebrew/brew/blob/8a807dcd7a95df4dd37c1d319603a6851501b38a/bin/brew#L24-L32

Because `LD_LIBRARY_PATH` already breaks `brew`, we should unset it.

Reply:
> We can unset it, but even invoking this script is already broken when `LD_LIBRARY_PATH` is set since it has `#!/bin/bash`.

Yes, can confirm that `LD_LIBRARY_PATH` breaks shebang `#!/bin/bash`.